### ISSUE TITLE
opengear.om.om_ports fails with unsupported paramaters error

### ISSUE THREAD

**Issue by kd9cpb** [opengear/opengear.om#15]
Currently we've been using ogcli to configure OM2200 serial ports as shown in the following example:
```
ogcli replace port ports-47 << 'END'
    baudrate="9600"
    databits="8"
    label="sw-testbed-nxos-1"
    mode="consoleServer"
    parity="none"
    pinout="X2"
    stopbits="1"
    escape_char="~"
    logging_level="disabled"
END
```

I'd like to stop using ogcli for this and start using the following in an Ansible playbook:
```
- name: Serial ports for og-testbed
  hosts: og-testbed
  collections:
    - opengear.om
  vars_files:
    - credentials.yml
  gather_facts: false
  connection: httpapi
  tasks:
  - name: Modify ports
    opengear.om.om_ports:
      config:
        ports:
          - id: 'ports-47'
            label: 'label test'
            databits: '8'
            mode: 'consoleServer'
            parity: 'none'
            pinout: 'X2'
            stopbits: '1'
            escape_char: '~'
            logging_level: 'disabled'
            baudrate: '9600'
      state:
        overridden
```

When I try to run the playbook, I get the following:
```
[tcostello@example ansible]$ ansible-playbook -i hosts.ini serial-og-testbed.yml
[WARNING]: Invalid characters were found in group names but not replaced, use -vvvv to see details

PLAY [Serial ports for og-testbed] *************************************************************************************

TASK [Modify ports] ****************************************************************************************************
fatal: [og-testbed.example.com]: FAILED! => {"changed": false, "msg": "Unsupported parameters for (basic.py) module: config.ports.available_baudrates, config.ports.raw_tcp, config.ports.single_session. Supported parameters include: baudrate, databits, escape_char, id, ip_alias, kernel_debug, label, logging_level, mode, name, parity, pinout, portnum, power, sessions, stopbits, terminal_emulation."}

PLAY RECAP *************************************************************************************************************
og-testbed.example.com     : ok=0    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0

```

After reviewing https://github.com/opengear/opengear.om/blob/master/playbooks/port_config.yaml and https://github.com/opengear/opengear.om/issues/8, I still can't seem to find a winning combination to match the known-good ogcli script. Could I either get some assistance troubleshooting that unsupported paramaters error, or possibly a known-good simple example playbook for opengear.om.om_ports?

**Comment by mattwit** [comment]
We were able to recreate your issue and looks like the argspec for ports needed to be updated. We made the changes and tested successfully.  A PR will be coming soon!

**Comment by mattwit** [comment]
Moving to closed. PR #16 merged.

**Comment by kd9cpb** [comment]
Can confirm the issue is resolved in 1.0.5, thanks so much!!!
